packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -8,18 +8,54 @@ import React, {
   useState,
 } from "react"
 import styled from "styled-components"
+import { system } from "styled-system"
 import { useCursor } from "use-cursor"
 import { ChevronIcon } from "../../svgs"
 import { SpacingUnit } from "../../Theme"
 import { useUpdateEffect } from "../../utils/useUpdateEffect"
 import { Box, BoxProps } from "../Box"
 import { Skip } from "../Skip"
 import { VisuallyHidden } from "../VisuallyHidden"
-import { CarouselNext, CarouselPrevious } from "./CarouselNavigation"
+import {
+  CarouselNavigationProps,
+  CarouselNext,
+  CarouselPrevious,
+} from "./CarouselNavigation"
 import { paginateCarousel } from "./paginate"
 
 const RAIL_TRANSITION_MS = 500
 
+const transition = system({ transition: true })
```

Comment:
why `transition: boolean` here when it expects a string as a prop?

Reply:
![](http://static.damonzucconi.com/_capture/PsYKTyQi8oLZ.png)

Reply:
nice